packages/components/psammead-section-label/src/__snapshots__/index.test.jsx.snap

```diff
@@ -1821,44 +1841,47 @@ exports[`SectionLabel Without bar With linking title should render correctly 1`]
 <div
   class="c0"
 >
-  <h2
+  <div
     class="c1"
+  />
+  <strong
+    class="c2"
   >
     <a
       aria-labelledby="test-section-label"
-      class="c2"
+      class="c3"
       href="/igbo/other-index"
     >
       <span
-        class="c3"
+        class="c4"
       >
         <span
-          class="c4"
+          class="c5"
           role="text"
         >
           <span
-            class="c5"
+            class="c6"
             dir="ltr"
             id="test-section-label"
           >
             This is text in a SectionLabel.
           </span>
           <span
             aria-hidden="true"
-            class="c6"
+            class="c7"
             dir="ltr"
           >
             See All
```

Comment:
If the journalist inputs a link - we won't want this in the recommendations component... will it be possible for a journalist to input a link?

Reply:
@greenc05 in the recommendations container in simorgh, it will be hard coded to not have a link text or an href, so we never have this outputted as a link

Reply:
Thanks @rhenshaw56 maybe we don't need tests for this scenario then?